db/schema.rb

```diff
@@ -322,6 +322,7 @@
     t.boolean "working_hours_enabled", default: false
     t.string "out_of_office_message"
     t.string "timezone", default: "UTC"
+    t.boolean "enable_email_collect"
```

Comment:
seems like the `default: true` isn't captured in the schema. Let's rerun the migration and ensure that is captured.